### PR TITLE
Add popups, some styles, fit bounds

### PR DESCRIPTION
**Pull request by luqmaan**
See http://open-austin.github.io/austingreenmap/

### CODE REVIEW

File: client/js/utils/ajax.js [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
 
 
-// FIXME: replace with XMLHTTPRequest
-
 export default function ajax(options) {
     var deferred = when.defer();
 
-    options.baseUrl = location.href;
-    options.json = true;
+    options.json = 'true';
```
Comment (luqmaan) [inline client/js/utils/ajax.js:8]: Kill this

File: client/js/utils/api.js [hunk 2]
```diff
@@ -8,25 +8,28 @@ var _cache = {};
...
+        var url = 'data/parks.json';
 
         if (_cache[url]) {
             return when.resolve(_cache[url]);
         }
 
-        return ajax({uri: url})
-            .tap((body) => _cache[url] = body);
+        return ajax({url: url})
+            .tap((data) => console.log(data))
```
Comment (johntyree) [inline client/js/utils/api.js:18]: Debugging?

File: client/js/components/ParkMap.jsx [hunk 3]
```diff
@@ -1,28 +1,67 @@
...
-                <button type="button" onClick={() => this.logMap()}>
-                    Show map in console
-                </button>
-                <Map id='map' center={this.props.center} zoom={15} ref="map">
+        return (
+            <div>
+                <div className='row'>
+                    <h3>{this.props.name}</h3>
+                </div>
+                <Map
```
Comment (johntyree) [inline client/js/components/ParkMap.jsx]: why is this split across five lines?

File: client/js/utils/api.js (hunk 2 above)
Comment (johntyree) [inline client/js/utils/api.js:18]: Need to sit down with paper and prove it, but it smells like `Promise` forms a Monad with `tap` as `join` :)

**Comment by luqmaan** [comment]
Uwotm8

**Comment by johntyree** [comment]
Looks fine.

File: client/js/components/ParkMap.jsx [hunk 4]
```diff
@@ -1,28 +1,67 @@
-import $ from 'jquery';
 import React from 'react';
 import { GeoJson, Map, Marker, Popup, TileLayer } from 'react-leaflet';
+import turf from 'turf';
+
+window.turf = turf;
+
+
+function onEachFacility(feature, layer) {
+    if (feature.properties && feature.properties) {
```
Comment (aaronmgdr) [inline client/js/components/ParkMap.jsx]: am i crazy or is this checking the truthiness of `feature.properties` twice?

Comment (luqmaan) [inline client/js/components/ParkMap.jsx]: Yep. :hankey:

File: client/js/utils/api.js (hunk 2 above)
Comment (luqmaan) [inline client/js/utils/api.js:18]: Hmm, I should ask what you mean by this at the next haskell meetup.

File: client/js/components/ParkMap.jsx (hunk 3 above)
Comment (luqmaan) [inline client/js/components/ParkMap.jsx]: Good catch.

**Pull request merged**